nibabel/tests/test_nifti1.py

```diff
@@ -132,40 +132,40 @@ def test_slope_inter(self):
         HDE = HeaderDataError
         assert_equal(hdr.get_slope_inter(), (1.0, 0.0))
         for in_tup, exp_err, out_tup, raw_values in (
-            # Null scalings
-            ((None, None), None, (None, None), (nan, nan)),
-            ((nan, None), None, (None, None), (nan, nan)),
-            ((None, nan), None, (None, None), (nan, nan)),
-            ((nan, nan), None, (None, None), (nan, nan)),
-            # Can only be one null
-            ((None, 0), HDE, (None, None), (nan, 0)),
-            ((nan, 0), HDE, (None, None), (nan, 0)),
-            ((1, None), HDE, (None, None), (1, nan)),
-            ((1, nan), HDE, (None, None), (1, nan)),
-            # Bad slope plus anything generates an error
-            ((0, 0), HDE, (None, None), (0, 0)),
-            ((0, None), HDE, (None, None), (0, nan)),
-            ((0, nan), HDE, (None, None), (0, nan)),
-            ((0, inf), HDE, (None, None), (0, inf)),
-            ((0, minf), HDE, (None, None), (0, minf)),
-            ((inf, 0), HDE, (None, None), (inf, 0)),
-            ((inf, None), HDE, (None, None), (inf, nan)),
-            ((inf, nan), HDE, (None, None), (inf, nan)),
-            ((inf, inf), HDE, (None, None), (inf, inf)),
-            ((inf, minf), HDE, (None, None), (inf, minf)),
-            ((minf, 0), HDE, (None, None), (minf, 0)),
-            ((minf, None), HDE, (None, None), (minf, nan)),
-            ((minf, nan), HDE, (None, None), (minf, nan)),
-            ((minf, inf), HDE, (None, None), (minf, inf)),
-            ((minf, minf), HDE, (None, None), (minf, minf)),
-            # Good slope and bad intercept generates error for get_slope_inter
-            ((2, None), HDE, HDE, (2, nan)),
-            ((2, nan), HDE, HDE, (2, nan)),
-            ((2, inf), HDE, HDE, (2, inf)),
-            ((2, minf), HDE, HDE, (2, minf))):
-            # Good slope and inter - you guessed it
-            ((2, 0), None, (None, None), (2, 0)),
-            ((2, 1), None, (None, None), (2, 1)),
```

Comment:
The colon/comma swap was the bug. Last two conditions weren't iterated over but were unassigned values in the loop body.

Updated the values in the unrun tests to not fail, but this should be checked for validity.